intel_extension_for_transformers/llm/runtime/graph/core/ne.h

```diff
@@ -40,7 +40,7 @@
 #define NE_FILE_VERSION 1
 
 #define NE_MAX_DIMS 4
-#define NE_MAX_NODES 4096
+#define NE_MAX_NODES 8192
```

Comment:
which model needs more nodes

Reply:
multi-requests inference of `gpt-j`  (`kv cache cpy`)